### ISSUE TITLE
Unexpected EOF while looking for matching `''

### ISSUE THREAD

**Issue by Epse** [andykais/bitwarden-dmenu#2]
**Describe the bug**
After logging in using the `bw` command, I run dmenu (with the options from the readme) from the command line. (Usually it is run from i3, but I wanted debug info).

It correctly prompts for my password, but after pressing Enter it errors out.

    /bin/sh: -c: line 0: unexpected EOF while looking for matching `''
    /bin/sh: -c: line 1: syntax error: unexpected end of file
    Error
        at module.exports.args (/usr/lib/node_modules/bitwarden-dmenu/src/exec-bitwarden  - 
    cli.js:13:11)
        at getSessionVar (/usr/lib/node_modules/bitwarden-dmenu/src/index.js:24:23)
    bitwarden is locked.
    clipboard is cleared

**To Reproduce**
Use a password that contains a single quote `'`. This will cause the error, correct password or not. Double quotes work fine, typing a backslash in front of a single quote (to escape it) doesn't fix the issue. This can be reproduced by typing any random password containing a single quote.

When typing a password containing two single quotes, the error does not happen. However I don't know what password is actually passed on to bitwarden.

**Expected behavior**
My password gets accepted and I can select an accout to copy the password from.

I suppose we would have to escape passwords before sending them in.

**Comment by andykais** [comment]
hey there! I tracked down this bug and actually the only character that needs to be escaped is the `'` character. The password was previously inside single quotes, which do not allow any escaped chars, so I switched to `$'password'` format which allows a few characters including the single quote to be escaped ([see man sh Quoting](https://linux.die.net/man/1/sh)).

tl;dr; this is fixed at commit https://github.com/andykais/bitwarden-dmenu/commit/a361ae9c07b11d80b24491480b49eddc37e1611d  and `bitwarden-dmenu@1.1.4`
